### ISSUE TITLE
Not able to build using plain source code without git.

### ISSUE THREAD

**Issue by nandbarkin** [vmware/differential-datalog#966]
I downloaded the source code from https://github.com/vmware/differential-datalog/releases/tag/v0.39.0 and attempted to build the binaries by following these steps: https://github.com/vmware/differential-datalog#installing-dependencies-manually

However when I ran this command 
`stack build`

It failed with following error:
```
<base-directory>/workspace/differential-datalog-0.39.0/src/Language/DifferentialDatalog/Version.hs:38:18: error:
    • Exception when trying to run compile-time code:
        GHEGitRunFailed "." ["rev-parse","--show-toplevel"] (ExitFailure 128) "" "fatal: not a git repository (or any of the parent directories): .git\n"
      Code: tGitInfoCwd
    • In the Template Haskell splice $$tGitInfoCwd
      In the first argument of ‘giHash’, namely ‘$$tGitInfoCwd’
      In the expression: giHash $$tGitInfoCwd
   |
38 | gitHash = giHash $$tGitInfoCwd
   |                  ^^^^^^^^^^^^^


--  While building package differential-datalog-0.39.0 (scroll up to its section to see the error) using:
      /Users/nandb/.stack/setup-exe-cache/x86_64-osx/Cabal-simple_mPHDZzAJ_3.0.1.0_ghc-8.8.4 --builddir=.stack-work/dist/x86_64-osx/Cabal-3.0.1.0 build lib:differential-datalog exe:ddlog exe:debugparser exe:ovsdb2ddlog --ghc-options " -fdiagnostics-color=always"
    Process exited with code: ExitFailure 1
```

The same failure is not observed when built from ddlog git repository

Machine details
OS: Mac OS catalina

**Comment by ryzhyk** [comment]
Interesting.  Looks like you need to checkout the tag from the git repo rather than using the archived code.  This is a bug, the code should compile outside of git.

**Comment by ryzhyk** [comment]
Closed via #986
